crates/nu-command/src/experimental/job_spawn.rs

```diff
@@ -75,10 +74,20 @@ impl Command for JobSpawn {
         let jobs = job_state.jobs.clone();
         let mut jobs = jobs.lock().expect("jobs lock is poisoned!");
 
+        let (send, recv) = mpsc::channel();
```

Comment:
The inner channel pair inside a job are `Sender<Value>, Receiver<Value>`;  Is it possible to make them `Sender<PipelineData>, Receiver<PipelineData>`?